main.go

```diff
@@ -18,18 +19,18 @@ var (
 	redisAlias       = flag.String("redis.alias", getEnv("REDIS_ALIAS", ""), "Redis instance alias for one or more redis nodes, separated by separator")
-	namespace        = flag.String("namespace", "redis", "Namespace for metrics")
-	checkKeys        = flag.String("check-keys", "", "Comma separated list of key-patterns to export value and length/size, searched for with SCAN")
-	checkSingleKeys  = flag.String("check-single-keys", "", "Comma separated list of single keys to export value and length/size")
-	scriptPath       = flag.String("script", "", "Path to Lua Redis script for collecting extra metrics")
-	separator        = flag.String("separator", ",", "separator used to split redis.addr, redis.password and redis.alias into several elements.")
-	listenAddress    = flag.String("web.listen-address", ":9121", "Address to listen on for web interface and telemetry.")
-	metricPath       = flag.String("web.telemetry-path", "/metrics", "Path under which to expose metrics.")
-	isDebug          = flag.Bool("debug", false, "Output verbose debug information")
-	logFormat        = flag.String("log-format", "txt", "Log format, valid options are txt and json")
-	showVersion      = flag.Bool("version", false, "Show version information and exit")
-	useCfBindings    = flag.Bool("use-cf-bindings", false, "Use Cloud Foundry service bindings")
-	redisMetricsOnly = flag.Bool("redis-only-metrics", false, "Whether to export go runtime metrics also")
+	namespace        = flag.String("namespace", getEnv("REDIS_EXPORTER_NAMESPACE", "redis"), "Namespace for metrics")
+	checkKeys        = flag.String("check-keys", getEnv("REDIS_EXPORTER_CHECK_KEYS", ""), "Comma separated list of key-patterns to export value and length/size, searched for with SCAN")
+	checkSingleKeys  = flag.String("check-single-keys", getEnv("REDIS_EXPORTER_CHECK_SINGLE_KEYS", ""), "Comma separated list of single keys to export value and length/size")
+	scriptPath       = flag.String("script", getEnv("REDIS_EXPORTER_SCRIPT", ""), "Path to Lua Redis script for collecting extra metrics")
+	separator        = flag.String("separator", getEnv("REDIS_EXPORTER_SEPARATOR", ","), "separator used to split redis.addr, redis.password and redis.alias into several elements.")
+	listenAddress    = flag.String("web.listen-address", getEnv("REDIS_EXPORTER_WEB_LISTEN_ADDRESS", ":9121"), "Address to listen on for web interface and telemetry.")
+	metricPath       = flag.String("web.telemetry-path", getEnv("REDIS_EXPORTER_WEB_TELEMETRY_PATH", "/metrics"), "Path under which to expose metrics.")
+	isDebug          = flag.Bool("debug", getEnvBool("REDIS_EXPORTER_DEBUG"), "Output verbose debug information")
+	logFormat        = flag.String("log-format", getEnv("REDIS_EXPORTER_LOG_FORMAT", "txt"), "Log format, valid options are txt and json")
+	showVersion      = flag.Bool("version", getEnvBool("REDIS_EXPORTER_VERSION"), "Show version information and exit")
```

Comment:
I don't think this one makes sense, can you remove it please?

Reply:
Done :)